src/main/java/nl/rijksoverheid/mev/common/util/BSNValidator.java

```diff
@@ -14,10 +14,34 @@ public final class BSNValidator {
     private static final int MIN_BSN_LEN = 8;
     private static final int ELF_PROEF_DENOMINATOR = 11;
 
+    /**
+     * Valideer één of meerdere bsns
+     *
+     * @param bsns de lijst met bsns
+     * @return of de bsns valide zijn
+     * @throws GezagException wanneer één van de bsns niet voldoet aan de elf
+     * proef
+     */
+    public boolean isValid(final List<String> bsns) {
+        boolean valid = true;
+
+        if (bsns != null) {
+            for (String bsn : bsns) {
+                valid = isValid(bsn);
+
+                if (!valid) {
+                    break;
+                }
+            }
+        }
+
+        return valid;
```

Comment:
This method can be simplified to one line with the Stream API: `bsns.stream().anyMatch(this::isValid);`

Reply:
denk dat je allMatch bedoelt? is aangepast